G/GoogleTest/build_tarballs.jl

```diff
@@ -28,6 +34,9 @@ make install
 # platforms are passed in on the command line
 platforms = expand_cxxstring_abis(supported_platforms())
 
+# Only CXX 11 is supported
+filter!(x -> cxxstring_abi(x) != "cxx03", platforms)
```

Comment:
What do you mean?

Reply:
https://github.com/google/googletest#supported-platforms
> GoogleTest requires a codebase and compiler compliant with the C++11 standard or newer.

And the CXX03 builds all fail

Reply:
Did you read https://docs.binarybuilder.org/stable/building/#Expanding-C-string-ABIs-or-libgfortran-versions and in particular (emphasis not mine)

> This ABI does _not_ have to do with the C++ standard used by the source code

?

Reply:
Gotcha, thanks.